README.md

```diff
@@ -15,22 +15,18 @@ First, clone the fluo-dev repo on a local disk with enough space to run Hadoop,
 git clone git@github.com:fluo-io/fluo-dev.git
 ```
 
-Next, download the binary tarballs of Hadoop 2.6.0, Accumulo 1.6.1 and Zookeeper 3.4.6
-and untar them in the `software/` directory of fluo-dev:
-```
-cd fluo-dev/software
-tar xzf zookeeper-3.4.6.tar.gz
-tar xzf accumulo-1.6.1-bin.tar.gz
-tar xzf hadoop-2.6.0.tar.gz
-``` 
-
-Copy and modify env.sh for your environment
+Create `env.sh` from the example.  This file is used to configure fluo-dev for your enviornment.
 ```
 cd conf/
 cp env.sh.example env.sh
 vim env.sh
 ```
 
+If you have not already, clone the Fluo repo.  Set FLUO_REPO in your env.sh to this location.
+```
+git clone git@github.com:fluo-io/fluo.git
```

Comment:
Not positive, but I think cloning that way uses ssh and requires a GH account.  Could use https for cloning which would not require a GH account.